src/lib/integration_test_lib/dune

```diff
@@ -2,5 +2,5 @@
   (public_name integration_test_lib)
   (name integration_test_lib)
   (inline_tests)
-  (preprocess (pps ppx_coda ppx_version ppx_optcomp ppx_let ppx_inline_test ppx_custom_printf))
+  (preprocess (pps ppx_coda ppx_version ppx_optcomp ppx_let ppx_inline_test ppx_custom_printf ppx_deriving.std ppx_assert ppx_sexp_conv ppx_compare))
```

Comment:
Because you don't directly depend on `ppx_optcomp` to do any constant value injection, you should be able to replace this whole preprocess definition with `(preprocess (pps ppx_coda ppx_version ppx_jane ppx_deriving.std))`